src/dlls/mscoree/coreclr/CMakeLists.txt

```diff
@@ -162,11 +162,18 @@ if(WIN32)
         clr_unknown_arch()
     endif()
 
+    set(DACTABLEGEN_EXE ${CMAKE_CURRENT_BINARY_DIR}/../../../ToolBox/SOS/DacTableGen/dactablegen.exe)
```

Comment:
A nit - Instead of this deep relative path, the following should work: ${CMAKE_BINARY_DIR}/src/ToolBox/SOS/DacTableGen/dactablegen.exe

Reply:
I did not like the ugly relative path, I will give that a try.